tests/e2e.sh

```diff
@@ -14,20 +14,19 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# Print commands
+set -x
+
 # Local vars
 ROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )
 ARGS=(-alsologtostderr -test.v -v 2)
-TESTARGS="${@}"
```

Comment:
Where are you puting the "$@"

Reply:
Started at line 106. We parse all inputs, and filter out the flags defined in this script not being passed down to go framework.

Reply:
ok